assets/src/design-system/components/dropDown/test/dropDown.js

```diff
@@ -0,0 +1,391 @@
+//   expect(menu).toBeInTheDocument();
+
+//   const menuItems = getAllByRole('option');
+//   expect(menuItems).toHaveLength(12);
+
+//   // wait for debounced callback to allow a select click handler to process
+//   jest.runOnlyPendingTimers();
+
+//   // focus first element in menu
+//   act(() => {
+//     fireEvent.keyDown(menu, {
+//       key: 'ArrowDown',
+//     });
+//   });
+
+//   expect(menuItems[0]).toHaveFocus();
+
+//   // focus second element in menu
+//   act(() => {
+//     fireEvent.keyDown(menu, {
+//       key: 'ArrowDown',
+//     });
+//   });
+//   expect(menuItems[1]).toHaveFocus();
+
+//   act(() => {
+//     fireEvent.keyDown(menu, { key: 'Enter' });
+//   });
+
+//   // The second item in the menu.
+//   // first prop we get back is the event
+//   expect(onClickMock).toHaveBeenCalledWith(
+//     expect.anything(),
+//     basicDropDownOptions[1].value
+//   );
+
+//   expect(onClickMock).toHaveBeenCalledTimes(1);
+
+//   expect(menu).toBeInTheDocument();
+// });
+
+// it('should trigger onMenuItemClick when using the down arrow plus enter key and close menu', async () => {
+//   const onClickMock = jest.fn();
+
+//   const { getByRole } = await renderWithProviders(
+//     <DropDown
+//       emptyText={'No options available'}
+//       dropDownLabel={'label'}
+//       options={basicDropDownOptions}
+//       onMenuItemClick={onClickMock}
+//       selectedValue={basicDropDownOptions[0].value}
+//     />
+//   );
+//   const selectButton = getByRole('button');
+//   fireEvent.click(selectButton);
+
+//   const menu = getByRole('listbox');
+//   expect(menu).toBeInTheDocument();
+
+//   // focus first element in menu
+//   act(() => {
+//     fireEvent.keyDown(menu, {
+//       key: 'ArrowDown',
+//     });
+//   });
+
+//   // focus second element in menu
+//   act(() => {
+//     fireEvent.keyDown(menu, {
+//       key: 'ArrowDown',
+//     });
+//   });
+
+//   act(() => {
+//     fireEvent.keyDown(menu, { key: 'Enter' });
+//   });
+
+//   // The second item in the menu.
+//   // first prop we get back is the event
+//   expect(onClickMock).toHaveBeenCalledWith(
+//     expect.anything(),
+//     basicDropDownOptions[2].value
+//   );
+
+//   expect(onClickMock).toHaveBeenCalledTimes(1);
+
+//   await waitFor(() => expect(menu).not.toBeInTheDocument());
+// });
+
+// it('should trigger onDismissMenu when esc key is pressed', async () => {
+//   const wrapper = await renderWithProviders(
+//     <DropDown
+//       dropDownLabel={'label'}
+//       options={basicDropDownOptions}
+//       selectedValue={basicDropDownOptions[1].value}
+//     />
+//   );
+//   const select = wrapper.getByRole('button');
+//   fireEvent.click(select);
+
+//   const menu = wrapper.queryByRole('listbox');
+//   expect(menu).toBeInTheDocument();
+
+//   act(() => {
+//     fireEvent.keyDown(menu, {
+//       key: 'Escape',
+//     });
+//   });
+
+//   await waitFor(() => expect(menu).not.toBeInTheDocument());
+// });
```

Comment:
Let's either remove these, activate them, or move them to another file. If they should remain disabled, but not deleted let's file a ticket.

Reply:
I filed a ticket here: https://github.com/google/web-stories-wp/issues/5764 (listed in the comment at the top of the disabled tests, line 260)